db/app.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const app = express();
+const { getTopics, getApi } = require('./controllers/topicsAndApi.controller')
+const getArticlesById = require('./controllers/articles.controller')
+
+app.use(express.json());
+
+app.get('/api/topics', getTopics);
+
+app.get('/api', getApi);
+
+app.get('/api/articles/:article_id', getArticlesById);
+
+// error handling middlesware starts here:
```

Comment:
As these get used more and more, might be worth making them an independent file. Refer to the error handling notes for help.